### PR TITLE
[INGI2347] Add exam June 2019

### PR DESCRIPTION
**Pull request by MartinBraquet**

### CODE REVIEW

File: src/q8/secu-INGI2347/exam/2019/Juin/All/secu-INGI2347-exam-2019-Juin-All.tex [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+\usepackage{../../../../../../eplcode}
+\lstset{language={C}}
+
+\hypertitle{Computer System Security}{8}{INGI}{2347}{2019}{Juin}{All}
+{INFO and ELEC students 2019}
+{Ramin Sadre}
+
+\section{TLS}
+\begin{enumerate}
+    \item What other than the chain of trust must the client verify in the certificate ?
```
Comment (Jimvy) [inline src/q8/secu-INGI2347/exam/2019/Juin/All/secu-INGI2347-exam-2019-Juin-All.tex]:
```suggestion
    \item What other than the chain of trust must the client verify in the certificate? 
```

File: src/q8/secu-INGI2347/exam/2019/Juin/All/secu-INGI2347-exam-2019-Juin-All.tex [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+\hypertitle{Computer System Security}{8}{INGI}{2347}{2019}{Juin}{All}
+{INFO and ELEC students 2019}
+{Ramin Sadre}
+
+\section{TLS}
+\begin{enumerate}
+    \item What other than the chain of trust must the client verify in the certificate ? 
+    \item Explain the TLS key exchange (assume RSA), only the steps after the validation of the certificate. How is the certificate used in this process?
+    \item In a society, they use TLS for their private network, you also self-signed the authority certificate (like in the exercices) and then install a root certificate on each computer of the company. What are the security risk? What can you do to solve this problem?
```
Comment (Jimvy) [inline src/q8/secu-INGI2347/exam/2019/Juin/All/secu-INGI2347-exam-2019-Juin-All.tex]:
```suggestion
    \item In a society, they use TLS for their private network, you also self-signed the authority certificate (like in the exercises) and then install a root certificate on each computer of the company. What are the security risk? What can you do to solve this problem?
```

File: src/q8/secu-INGI2347/exam/2019/Juin/All/secu-INGI2347-exam-2019-Juin-All.tex [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+\section{Cryptography}
+
+\begin{enumerate}
+    \item You need to authenticate a communication and have at your disposal two different kinds of hash:
+    \begin{itemize}
+        \item Alice sends $(m,F(k,m))$
+        \item Alice sends $(m,H(k||m))$
+    \end{itemize}
+    where $F$ is the HMAC using SHA256 and $H$ is SHA256. Which one is the best to the integrity of the message ? Explain.
```
Comment (Jimvy) [inline src/q8/secu-INGI2347/exam/2019/Juin/All/secu-INGI2347-exam-2019-Juin-All.tex]:
```suggestion
    where $F$ is the HMAC using SHA256 and $H$ is SHA256. Which one is the best to the integrity of the message? Explain.
```

**Comment by MartinBraquet** [comment]
That's you all over.

**Pull request merged**